conf/arcc.config

```diff
@@ -0,0 +1,29 @@
+params {
+    config_profile_name = 'ARCC'
+    config_profile_description = 'Advanced Research Computing Center (ARCC) for the University of Wyoming'
+    config_profile_contact = 'Robert A. Petit III (@rpetit3)'
+    config_profile_url = 'http://www.uwyo.edu/arcc/'
+
+    // Custom params for ARCC
+    singularity_pull_docker_container = true
+
+    slurm_account = 'healthdatasci'
+    slurm_opts = ''
```

Comment:
What sort of things do you think would be going here? Isn't there a risk of a clash with existing opts that Slurm specifies by default?